src/test/java/org/example/SearchFieldTest.java

```diff
@@ -1,84 +1,50 @@
 package org.example;
 
-import org.openqa.selenium.WebDriver;
-import org.testng.annotations.Test;
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
-import org.testng.Assert;
-import org.testng.annotations.DataProvider;
-import java.time.Duration;
-import java.util.Iterator;
-
-import static org.testng.Assert.assertTrue;
+import org.testng.annotations.Test;
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
 
 public class SearchFieldTest {
+    String path = System.getProperty("user.dir");
 
     @Test
     public void incorrectInfo() {
-        System.setProperty("webdriver.chrome.driver", "C:\\Users\\y4ken\\Documents\\GitHub\\Epam-Selenium-Practice-Group-5\\src\\test\\resources\\webdriver\\chromedriver.exe");
-
-        WebDriver webDriver = new ChromeDriver();
+        System.setProperty("webdriver.chrome.driver", path+"\\resources\\webdriver\\chromedriver.exe");
```

Comment:
It is not necessary, in selenide library they use bonigarcia to work with webdriver